core/arch/riscv/kernel/tee_time_rdtime.c

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2022 NXP
+ */
+
+#include <kernel/clint.h>
+#include <kernel/tee_time.h>
+#include <kernel/time_source.h>
+#include <riscv.h>
+#include <utee_defines.h>
+
+static TEE_Result riscv_get_sys_time(TEE_Time *time)
+{
+	uint64_t _time;
+
+	_time = read_time();
```

Comment:
`uint64_t tm =  read_time();` because we want all variables initialized (coding rule), and the underscore is not really justified here (not like reserved names in user libraries).